index.js

```diff
@@ -130,10 +129,7 @@ client.on('message', async message => {
 			}
 			currency.add(message.author.id, -items.cost);
 			await user.addItem(items, message.author.id);
-
```

Comment:
unexepected modification on buy command, I'll let this one slip, don't let it happen again chief.